src/server/user-controller.js

```diff
@@ -411,6 +426,20 @@ class UserController {
       [needsPermission(permissions.EDIT_USERS), validateParams(userIdParamsSchema)],
       (req, res) => this.handleDeleteAllUserStorageReminders(req, res)
     );
+
+    router.post(
+      '/api/v1/users/favorites',
+      jsonParser,
+      validateBody(favoriteBodySchema),
+      (req, res) => this.handlePostFavorite(req, res)
```

Comment:
I think we should have some permission restriction here? If these endpoints would be called without a user, there would be an exception thrown through the `userService`, but I don't think it should reach that point.

Reply:
You're absolutely right. I went for `VIEW_DOCS` as it is more or less the most basic thing.